### DISCUSSION TITLE
How to determine which milestone was closed

### DISCUSSION THREAD

**Discussion by WyriHaximus** [community/community#25847] [Actions]
<p>Working on an action that will run when a milestone is closed. But I currently have no way to determine which milestone the action was triggered for. Unless I missed something in the documentation. Does anyone have pointers or a way to resolve this?</p>

**Comment by chrispat** [comment, marked as answer]
<p>Please see <a href="https://help.github.com/en/articles/events-that-trigger-workflows#milestone-event-milestone" rel="nofollow noopener">https://help.github.com/en/articles/events-that-trigger-workflows#milestone-event-milestone</a> for information on how to only trigger when the milestone is closed.</p>
<p>As part of the context you have access to the full event payload, for milestone you can find those details in the events and payloads documentation <a href="https://developer.github.com/v3/activity/events/types/#milestoneevent" rel="nofollow noopener">https://developer.github.com/v3/activity/events/types/#milestoneevent</a>.  To access the milestone title, for example, you could use ${{ github.event.milestone.title }}.  You can also access the payload on disk by reading the file located at $GITHUB_EVENT_PATH.</p>

**Comment by WyriHaximus** [comment]
<p>Cheers! Didn’t realise the entire event was available through <code>${{ github.event }}</code>. Thank you for your help :+1:</p>
